include/networkit/generators/DynamicGraphSource.hpp

```diff
@@ -74,7 +76,7 @@ class DynamicGraphSource {
     /**
      * Continue generating until the number of time steps reaches this upper limit.
      */
-    virtual void generateTimeSteps(count t);
+    virtual void TLX_DEPRECATED(generateTimeSteps(count t));
```

Comment:
Why is this deprecated now?

Reply:
I forgot to deprecate this before, the implementation is this
```
void DynamicGraphSource::generateTimeSteps(count t) {
     while (G->time() < t) {
         this->generate();
     }
     G->shrinkToFit();
  }
```
which is something that we cannot support anymore if `time` is removed from `Graph`.

Reply:
I think given the total amount of changes, it is totally fine to miss something - but IMHO I would remove the function directly. As of now it does nothing than compacting the graph, which is misleading.

Reply:
That's true, OTOH removing it might break some other people's code so probably deprecating it is the safest solution.

Reply:
Even though it is not interesting for this PR, but (a more general opinion):

I would prefer breaking code over unexpected behavior (and you don't know what caused it). Or at least mention here, that the function might not do what you think.